README.md

```diff
@@ -227,6 +257,30 @@ A list of function names that are valid as Testing Library custom renders. Relat
 
 [You can find more details here](docs/migrating-to-v4-guide.md#testing-librarycustom-renders).
 
+## Troubleshooting
+
+### There are errors reported in non-testing files
+
+If you find ESLint errors related to `eslint-plugin-testing-library` in files other than testing, this could be caused by [Aggressive Reporting](#aggressive-reporting).
+
+You can avoid this by:
+
+1. [running `eslint-plugin-testing-library` only against testing files](#run-the-plugin-only-against-test-files)
+2. [limiting the scope of Aggressive Reporting through Shared Settings](#shared-settings)
+
+If you think the error you are getting is not related to this at all, please [fill a new issue](https://github.com/testing-library/eslint-plugin-testing-library/issues/new/choose) with as many details as possible.
+
+### There are false positives in testing files
+
+If you are getting false positive ESLint errors in your testing files, this could be caused by [Aggressive Reporting](#aggressive-reporting).
+
+You can avoid this by:
+
+1. [running `eslint-plugin-testing-library` only against testing files](#run-the-plugin-only-against-test-files)
+2. [limiting the scope of Aggressive Reporting through Shared Settings](#shared-settings)
```

Comment:
We're already in testing files, so the only option is limiting the scope I think?

```suggestion
You can avoid this by [limiting the scope of Aggressive Reporting through Shared Settings](#shared-settings)
```

Reply:
We are not running the plugin only for testing files. We had a discussion around this when [I implemented the glob pattern for reporting only on testing files](https://github.com/testing-library/eslint-plugin-testing-library/pull/296#issuecomment-804216452), which finally [was discarded and the whole mechanism was reverted](https://github.com/testing-library/eslint-plugin-testing-library/pull/297) in favor leaving the user to decide which files to lint with proper ESLint config.

Reply:
In this case we are, hence the "There are false positives in testing files" title

https://github.com/testing-library/eslint-plugin-testing-library/pull/339/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R280